Automattic/jetpack
# js-packages: Fix textdomains

## Proposed changes:

We have a bunch of packages that, for historical reasons, use "jetpack" rather than a unique text domain. Fix them now.

### Other information:


## Jetpack product discussion



## Does this pull request change what data or activity we track or use?



## Testing instructions:






* Check the Composer-package and plugin artifacts. There should be no relevant changes, thanks to use of `@automattic/babel-plugin-replace-textdomain` when bundling.
  * The only change I noticed was that the move of a few translator comments from outside a `sprintf` to inside is reflected in the bundles.
